src/test/java/com/google/api/codegen/testdata/simplecompute_config.baseline

```diff
@@ -41,8 +41,8 @@ interfaces:
   # The entity_name is the name to be used as a basis for generated methods and
   # classes.
   collections:
-  - name_pattern: \{project}/regions/{region}/addresses
-    entity_name: region
+  - name_pattern: \{project}/regions/{region}
```

Comment:
The fundamental problem with not having the "projects/" literal in the name patterns in this client is that it is inconsistent with One Platform clients, which do include it in all resource names that start with project.

Reply:
Done. As a plus, the name patterns don't weirdly begin with the escaping backslash!